project_task_description_template/models/task_template.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import fields, models
+
+
+class TaskTemplate(models.Model):
```

Comment:
Please rename class like model name: `ProjectTaskTemplate`

Reply:
okay, I'll fix it